stats/thresholds.go

```diff
@@ -122,9 +213,9 @@ func (tc thresholdConfig) MarshalJSON() ([]byte, error) {
 
 // Thresholds is the combination of all Thresholds for a given metric
 type Thresholds struct {
-	Runtime    *goja.Runtime
 	Thresholds []*Threshold
 	Abort      bool
+	Sinked     map[string]float64
```

Comment:
One more thing: since `Sinked` is not used anywhere else, I suggest making it `sinked`.